info/release-notes/23.next.0-readme.txt

```diff
@@ -80,6 +80,12 @@ Improvements
   - Support macOS ARM platform    
     https://github.com/mytourbook/mytourbook/discussions/1174
 
+* Tour Blog View & Tour Analyzer View
+  - Refreshing each view when the measurement system is changed
+
+* Tour Print
+  - Added recorded and paused time values. Rounded values (elevation gain, distance...)
```

Comment:
You have not tagged your topics with your name

Reply:
Good catch, I forgot. Fixed